export const getFailedMessage = (countMistakes: number): string =>
  `Ooops... You have ${countMistakes} mistakes. You can try again!`;
